apps/simple-trading-app/src/app/components/simple-market-list/simple-market-toolbar.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import { t } from '@vegaprotocol/react-helpers';
+import { theme } from '@vegaprotocol/tailwindcss-config';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Button, Select } from '@vegaprotocol/ui-toolkit';
+import useMarketFiltersData from '../../hooks/use-markets-filter';
+import { STATES_FILTER } from './constants';
+
+const SimpleMarketToolbar = () => {
+  const navigate = useNavigate();
+  const params = useParams();
+  const { assets, products, assetsPerProduct } = useMarketFiltersData();
+  const [activeNumber, setActiveNumber] = useState(
+    products?.length ? products.indexOf(params.product || '') + 1 : -1
+  );
+  const [activeAsset, setActiveAsset] = useState(params.asset || 'all');
+  const [activeState, setActiveState] = useState(params.state || 'Active');
+  const [sliderStyles, setSliderStyles] = useState<Record<string, string>>({});
+  const slideContRef = useRef<HTMLUListElement | null>(null);
+  const onMenuClick = useCallback(
```

Comment:
chore (non-blocking): wrapping this in a useCallback seems redundant here

Reply:
sure, good catch, removed